ci/Jenkinsfiles/build.groovy

```diff
@@ -100,7 +100,12 @@ pipeline {
                 Compile
                 ----------------------------------------"""
                 echo "MAVEN_OPTS=$MAVEN_OPTS"
-                sh 'mvn -B -nsu -T4C install -DskipTests'
+                sh """
+                  mvn -B -nsu -T4C install -DskipTests \
+                    -Dnode.server.id=nexus-internal \
+                    -Dnode.download.root=https://nexus.platform.dev.nuxeo.com/repository/nodejs-upstream/ \
+                    -Dnode.npm.download.root=https://nexus.platform.dev.nuxeo.com/repository/npmjs-registry/
```

Comment:
It would be interesting to get these URLs from the Jenkins environment, in a similar way we do for Docker registry?

Reply:
Also, shouldn't we do something for the ftest step that directly calls npm?

Reply:
> Also, shouldn't we do something for the ftest step that directly calls npm?

I don't think so.
During the `Compile` stage, node and npm are installed in `nuxeo-coldstorage-web/node` directory and any further call to `npm` in `nuxeo-coldstorage-web` dir will leverage this installation along with its configuration.